Dockerfile

```diff
@@ -67,12 +73,9 @@ RUN gem update bundler && \
 
 COPY --chown=essi:essi . .
 
-# The defaults for ImageMagick are too constrained, override so that MiniMagick won't fail
```

Comment:
Does the new ImageMagick version installed for iiif_print have policy defaults above what this was intended to address?

Reply:
The policies seem to be using the available resource strategy to set limits. This is what is reported from within the container for the test instance, which is a lot bigger than the explicit values we had set:
```
convert -list resource

Resource limits:
  Width: 461.169PP
  Height: 461.169PP
  Area: 134.527GP
  List length: unlimited
  Memory: 62.6438GiB
  Map: 125.288GiB
  Disk: unlimited
  File: 12288
  Thread: 4
  Throttle: 0
  Time: unlimited
  ```